### PR TITLE
Add rawkvclient with context parameters

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

Step 1 of https://github.com/tikv/client-go/issues/200

### CODE REVIEW

**Review by sticnarf — APPROVED** [review]

File: rawkv/rawkv.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+// ClusterID returns the TiKV cluster ID.
+func (c *Client) ClusterID() uint64 {
+	return c.client.ClusterID()
+}
+
+// Get queries value with the key. When the key does not exist, it returns `nil, nil`.
+func (c *Client) Get(ctx context.Context, key []byte) ([]byte, error) {
+	return c.client.Get(key)
```
Comment (iosmanthus) [inline rawkv/rawkv.go:70]: The `ctx` is not used in the function. How can user cancel this execution using this `ctx`?

Comment (sticnarf) [reply rawkv/rawkv.go:70]: I think it's remained for future work...

Comment (iosmanthus) [reply rawkv/rawkv.go:70]: Well then...

Comment (iosmanthus) [reply rawkv/rawkv.go:70]: Could we add some TODO to the function signatures in case we forget to refactor these codes? @disksing

Comment (disksing) [reply rawkv/rawkv.go:70]: won't forget but I think it is better to add TODO :)

**Pull request merged**